config/pure-python-without-pypy/MANIFEST.in

```diff
@@ -6,6 +6,7 @@ include buildout.cfg
 include tox.ini
 
 exclude MANIFEST.in
+exclude .meta.cfg
```

Comment:
I think we should ignore it via `setup.cfg`. Otherwise you get a warning on installation time that `.meta.cfg` does not exist.

Reply:
Done.